src/psyclone/configuration.py

```diff
@@ -945,18 +946,20 @@ def make_property(self, dereference_format, type_name):
 
         :raises InternalError: if type_name is not 'scalar' or 'array'
         '''
-        if not type_name in ['array', 'scalar']:
+        if type_name not in ['array', 'scalar']:
             from psyclone.errors import InternalError
             raise InternalError("Type must be 'array' or 'scalar' but is "
                                 "'{0}'.".format(type_name))
 
-        return self._property(dereference_format, type_name)
+        Property = namedtuple("Property", "fortran type")
+        return Property(dereference_format, type_name)
 
     # ---------------------------------------------------------------------
     @property
     def grid_properties(self):
-        ''':returns: the dictionary containing the grid properties.
-        :rtype: a dictionary with values of pairs (dereference-format, type)
+        ''':returns: the dict containing the grid properties.
+        :rtype: a dict with values of namedtuple("Property","fortran type")
```

Comment:
You're missing a line-continuation char here.

Reply:
Fixed.